ui/app/components/app/home-notification/home-notification.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import HomeNotification from './home-notification.component'
+
+export default {
+  title: 'HomeNotification',
+}
+
+export const Standard = () => (
+  <HomeNotification
+    acceptText="Accept"
+    onAccept={() => {}}
```

Comment:
Nit: Generally for function props that perform some action, it's better to use an `action` from '@storybook/addon-actions'. That way it'll be logged in the Actions tab in storybook when the button is clicked, so you can see that the button works (and what args are passed in)